charts/postgresql/templates/postgresql-sfagent-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "postgresql.fullname" . }}-sfagent
+  labels:
+    {{- include "postgresql.labels" . | nindent 4 }}
+
+data:
+  config.yaml: |-
+    key: "{{ .Values.global.key }}"
+    metrics:
+        plugins:
+        - name: postgres
+          enabled: true
+          interval: 30
+          config:
+            port: 5432
+            host: 127.0.0.1
+            user: postgres
+            password: postgres
```

Comment:
change this to
user: {{ .Values.rootUser}}
password: {{ .Values.rootPassword }}